src/course-unit/course-xblock/xblock-content/iframe-wrapper/iframe-wrapper.js

```diff
@@ -17,14 +17,31 @@ import {
  *                   Only required for legacy XBlocks that don't declare their
  *                   JS and CSS dependencies properly.
  * @param type The XBlock's type (openassessment, discussion, video, etc.)
+ * @param stylesWithContent Custom CSS global for all XBlock's.
  */
-export default function wrapBlockHtmlForIFrame(html, sourceResources, studioBaseUrl, type) {
+export default function wrapBlockHtmlForIFrame(
+  html,
+  sourceResources,
+  studioBaseUrl,
+  type,
+  stylesWithContent,
+) {
   const resources = normalizeResources(sourceResources);
 
   /* Extract CSS resources. */
   const cssUrls = filterAndExtractResources(resources, 'url', 'text/css');
   const sheets = filterAndExtractResources(resources, 'text', 'text/css');
 
+  let additionalCSSTags = '';
```

Comment:
```suggestion
  let additionalCssTags = '';
```
[nit] I remember on the Paragon we had a linter that didn't like when we did something like `CSS` in the middle of a variable name. The idea is to follow `camelCase` when it is possible. When we do `CSS` it is kind of small `camelCase` violation.

Reply:
Ok, corrected